xmlsec will fail pkgconf test pipeline

If the pkgconf test pipeline is added to xmlsec it will fail in the following way:

```
2024/11/12 23:16:13 INFO running step "test/pkgconf"
2024/11/12 23:16:13 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:16:13 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:16:13 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:16:13 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:16:13 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + basename /home/build/melange-out/xmlsec-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + dev_pkg=xmlsec-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + apk info -L xmlsec-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + grep -q ^Name: usr/lib/pkgconfig/xmlsec1-openssl.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + basename usr/lib/pkgconfig/xmlsec1-openssl.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + lib_name=xmlsec1-openssl uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + echo usr/lib/pkgconfig/xmlsec1-openssl.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + grep -q '^usr/lib/pkgconfig/xmlsec1-openssl.pc$\|^usr/share/pkgconfig/xmlsec1-openssl.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + pkgconf --exists xmlsec1-openssl uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN + pkgconf --print-errors --exists xmlsec1-openssl uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN Package libxml-2.0 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN Perhaps you should add the directory containing `libxml-2.0.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN Package 'libxml-2.0', required by 'xmlsec1-openssl', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 WARN Package 'libxslt', required by 'xmlsec1-openssl', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:13 INFO pod 4ebe19166fc6d28fcb9cd8c7615e0bbab953d24c55aab94b9b4c1a38e4ea70da terminated
2024/11/12 23:16:13 INFO pod 0f2228912b77eea2c9babbd7a4286806f0357a13a016067c34e1359b53bac9f2 terminated
2024/11/12 23:16:13 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:16:13 INFO   workspace dir: /home/user/tmp/melange-workspace-1537796718
2024/11/12 23:16:13 INFO   guest dir: /home/user/tmp/melange-guest-1440958756
2024/11/12 23:16:13 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/xmlsec] Error 1
```

--- a/xmlsec.yaml
+++ b/xmlsec.yaml
@@ -1,7 +1,7 @@
 package:
   name: xmlsec
   version: 1.3.4
-  epoch: 1
+  epoch: 2
   description: C based implementation for XML Signature Syntax and Processing and XML Encryption Syntax and Processing
   copyright:
     - license: MIT
@@ -83,9 +83,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libxml2-dev
+        - libxslt-dev
     description: xmlsec dev
     test:
       pipeline:
+        - uses: test/pkgconf
         - runs: |
             xmlsec1-config --version
             xmlsec1-config --help

